underfs/local/src/main/java/alluxio/underfs/local/LocalUnderFileSystem.java

```diff
@@ -293,7 +293,9 @@ public boolean mkdirs(String path, MkdirsOptions options) throws IOException {
         setMode(file.getPath(), options.getMode().toShort());
         FileUtils.setLocalDirStickyBit(file.getPath());
         try {
-          setOwner(file.getPath(), options.getOwner(), options.getGroup());
+          if (options.getOwner() != null || options.getGroup() != null) {
+            setOwner(file.getPath(), options.getOwner(), options.getGroup());
+          }
```

Comment:
Is this necessary or just for consistency?

Reply:
It is unnecessary. I removed the check.